### PR TITLE
Add Sokol bindings

### PR DESCRIPTION
**Pull request by deadwanderer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ List of Odin bindings to C/C++ libraries.
 | [Nuklear](https://github.com/cr15th0f3r/odin-nuklear) | Bindings for [Nuklear](https://github.com/Immediate-Mode-UI/Nuklear), s single-header ANSI C immediate mode cross-platform GUI library |  None | UI, ImGUI, Gamedev, Cross Platform |
 | [Box2C](https://gitlab.com/L-4/odin-box2c) | Idiomatic Odin bindings for Box2c (aka Box2D 3.0, not to be confused with older versions which were written in C++!) | None | Gamedev, Physics
 | [Box2C (Odin-style)](https://github.com/cr15th0f3r/odin-box2d) | Box2D 3.0 bindings for Odin language | None | Gamedev, Physics
-
+| [Sokol](https://github.com/floooh/sokol-odin/) | Odin bindings for [Sokol](https://github.com/floooh/sokol/), a collection of single-header C libraries for cross-platform windowing, graphics, audio, file loading, and more! | [zlib](https://github.com/floooh/sokol/blob/master/LICENSE) | Gamedev, Windows, Input, Audio, Cross-Platform
```
Comment (laytan) [inline README.md:164]: Windows and cross-platform seems a bit like opposite tags to me
```suggestion
| [Sokol](https://github.com/floooh/sokol-odin/) | Odin bindings for [Sokol](https://github.com/floooh/sokol/), a collection of single-header C libraries for cross-platform windowing, graphics, audio, file loading, and more! | [zlib](https://github.com/floooh/sokol/blob/master/LICENSE) | Gamedev, Input, Audio, Cross-Platform
```

Comment (jakubtomsu) [reply README.md:164]: I agree.

Also, Sokol could have a star (⭐) since it's very high quality

Comment (deadwanderer) [reply README.md:164]: Ah, I was thinking windowing, not Windows operating system...

**Comment by jakubtomsu** [comment]
nice, thank you!

**Pull request merged**